src/systems/filecoin_nodes/clock/_index.md

```diff
@@ -2,26 +2,29 @@
+Filecoin assumes weak clock synchrony amongst participants in the system. That is, the system relies on participants having access to a globally synchronized clock (tolerating some bounded drift).
+
+Filecoin relies on this system clock in order to secure consensus.  Specifically the clock is necessary to support validation rules that prevent block producers from mining blocks with a future timstamp, running leader elections more frequently than the protocol allows, and performing a class of misbehaviors involving withholding and rushing blocks.
+
 
 ## Clock uses
-Specifically, the Filecoin system clock is used:
+The Filecoin system clock is used:
 
-- to validate incoming blocks and ensure they were mined in the appropriate round, looking at the wall clock time in conjunction with the block's epoch number (see {{<sref block_validation>}}).
-- to help protocol convergence by giving miners a specific cutoff after which to reject incoming blocks in this round (see {{<sref chain_sync>}}).
-- to maintain protocol liveness by allowing participants to try leader election in the next round if no one has produced a block in this round (see {{<sref storage_power_consensus>}}).
+- by syncing nodes to validate that incoming blocks were mined in the appropriate epoch given their time stamp (see {{<sref block_validation>}}).  This is possible because the system clock maps all times to a unique epoch number totally determined by the start time in the genesis block.
+- by syncing nodes in the `CHAIN_FOLLOW` state to validate that incoming blocks were mined during the current epoch, not in the past or future.
+- by syncing nodes in the `CHAIN_FOLLOW` state to determine the cutoff period during each epoch after which they must reject incoming blocks for that epoch (see {{<sref chain_sync>}}).
+- by mining nodes to maintain protocol liveness by allowing participants to try leader election in the next round if no one has produced a block in the current round (see {{<sref storage_power_consensus>}}).
 
 In order to allow miners to do the above, the system clock must:
 
-1. have low clock drift: at most on the order of 1s (i.e. markedly lower than epoch time) at any given time.
-2. maintain accurate network time over many epochs: resyncing and enforcing accurate network time.
-3. set epoch number on client initialization equal to `epoch ~= (current_time - genesis_time) / epoch_time`
+1. Have low enough clock drift relative to other nodes so that blocks produced at and propagated from one node arrive within the same epoch boundary on the other node.  Maintain this throughout operation
+2. Set epoch number on node initialization equal to `epoch = Floor[(current_time - genesis_time) / epoch_time]`
```

Comment:
why the removal of 2.?

Reply:
old bullet 2 was refining and describing implementation of old bullet 1 so I consolidated.